README.rst

```diff
@@ -155,6 +155,15 @@ makes Vulture ignore the "greet" method::
     dead_code.py:8: unused variable 'message' (60% confidence)
 
 
+Exit Codes
+----------
+
+Vulture exits with an exit code of ``0`` when vulture is executed successfully
+and no dead code is found. In any other case, for ex. when vulture finds dead
+code or when vulture fails to execute successfully (perhaps because input file
+not found or corrput), vulture exits with an exit code of ``1``.
```

Comment:
People tend to skip long paragraphs. Therefore, I'd rather use the table from the corresponding issue.